spec/integration/util/windows/adsi_spec.rb

```diff
@@ -13,7 +13,7 @@
       #     HRESULT error code:0x800708ad
       #           The user name could not be found.
       # Matching on error code alone is sufficient
```

Comment:
But matching on only the error code doesn't match the intent described in lines 11-12 immediately above. Perhaps we need to pull the comments? Or rephrase?

Reply:
Good questions. This was here before the refactor -- https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/adsi.rb#L209-L224 and https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/adsi.rb#L439-L454

I don't want to rephrase comments made before this refactor here b/c this PR's really meant to purely move stuff around without changing the original intent.